starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,46 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+h1{
+  color: white;
+  background-image: url('/Users/Kristen/Desktop/lab-css-recipes-clone/starter-code/french-onion-soup-starter-code/images/soup.jpg');
+  background-repeat: no-repeat;
+  height: 500px;
+  width: 80%;
```

Comment:
with background size cover you would not need to set the width unless you want to make the image smaller